src/content/docs/en/guides/routing.mdx

```diff
@@ -494,6 +469,25 @@ interface Page<T = any> {
 }
 ```
 
+The following example template displays current information for the page along with links to navigate between pages:
```

Comment:
I'm not sure `template` is needed here... I feel like it makes the sentence heavier... (I don't know if it makes sense in English) but I'm not the best person to discuss the wording.
```suggestion
The following example displays current information for the page along with links to navigate between pages:
```